pkg/webhook/clusterpropagationpolicy/mutating.go

```diff
@@ -82,7 +83,7 @@ func (a *MutatingAdmission) Handle(_ context.Context, req admission.Request) adm
 		}
 	}
 
-	if util.GetLabelValue(policy.Labels, policyv1alpha1.ClusterPropagationPolicyPermanentIDLabel) == "" {
+	if req.Operation == admissionv1.Create {
```

Comment:
Will we be able to do this in the next version? For upgraded resources, it may no longer have the opportunity to experience creation events.

Reply:
Actually, all propagation policies are already labeled, at least in v1.9, so no problem with upgrading.

See the code here, which was introduced in v1.9-:
https://github.com/karmada-io/karmada/pull/4949/files#diff-fdb6430a46e82ace9f449b971619671eac711f7a82d6b862c130236a65cd5101L723-L734

Reply:
also applies to the release-1.8.